subliminal/providers/legendastv.py

```diff
@@ -164,6 +164,13 @@ class LegendasTVProvider(Provider):
     subtitle_class = LegendasTVSubtitle
 
     def __init__(self, username=None, password=None):
+
+        # Provider needs UNRAR installed. If not available raise ConfigurationError
+        try:
+            rarfile._check_unrar_tool()
```

Comment:
I don't think it actually raises `rarfile.RarCannotExec`.
See [markokr/rarfile@rarfile_3_0-`/rarfile.py#L2927-L2948`](https://github.com/markokr/rarfile/blob/rarfile_3_0/rarfile.py#L2927-L2948)

And a note, in the next version, it will return True or False.
See https://github.com/markokr/rarfile/pull/32